### PR TITLE
[IBCDPE-807] New PEGS Challenge workspace on tower-prod

### PR DESCRIPTION
**Pull request by jaymedina**
- [x] Create a new PEGS Challenge project workspace on tower-prod
- [x] Cost center is `12470001`
- [x] The following have been added with Read/Write access:
Rongrong Chai
Tom Yu
Jenny Medina
Brad MacDonald
Verena Chung
Gaia Andreoletti

### CODE REVIEW

File: config/projects-prod/pegs-challenge.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+stack_name: pegs-challenge
+
+stack_tags:
+  Department: DNT
+  Project: PEGS Challenge
+  OwnerEmail: nextflow-admins@sagebase.org
+  CostCenter: 12470001 # Valid values here: https://github.com/Sage-Bionetworks/aws-infra/tree/master/templates/tags
```
Comment (jaymedina) [inline config/projects-prod/pegs-challenge.yaml]: In some of the other examples, I see tags formatted like `cost-center-name / 123456` - is the cost center name necessary or can I leave it like this?

Comment (thomasyu888) [reply config/projects-prod/pegs-challenge.yaml]: Good catch, this needs to be modified. I posted in #sageit

Comment (thomasyu888) [reply config/projects-prod/pegs-challenge.yaml]: @jaymedina : "PEGS Gene Environment / 124700"

File: config/projects-prod/pegs-challenge.yaml
Comment (thomasyu888) [inline config/projects-prod/pegs-challenge.yaml:1]: Can this follow the same file naming structure as the other projects?

`pegs-challenge-project`?

I think the code will actually fail if it doesn't follow that structure

**Review by thomasyu888 — CHANGES_REQUESTED** [review]
Nice!  just left some comments

File: config/projects-prod/pegs-challenge.yaml (hunk 1 above)
Comment (jaymedina) [reply config/projects-prod/pegs-challenge.yaml]: Thanks for posting - I've updated accordingly

File: config/projects-prod/pegs-challenge.yaml
Comment (jaymedina) [reply config/projects-prod/pegs-challenge.yaml:1]: OK, noted for the future

File: config/projects-prod/pegs-challenge-project.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+stack_name: pegs-challenge
```
Comment (thomasyu888) [inline config/projects-prod/pegs-challenge-project.yaml]:
```suggestion
stack_name: pegs-challenge-project
```

**Review by thomasyu888 — APPROVED** [review]
✅  LGTM!

**Pull request merged**